src/krb5_common_plug.c

```diff
@@ -246,7 +245,7 @@ void des_cbc_mac_shishi(char key[8], char iv[8], unsigned char *in, size_t inlen
 	DES_key_schedule dks;
 #ifdef _MSC_VER
 	unsigned char *ct;
-	ct = mem_alloc(inlen);
+	ct = malloc(inlen);
```

Comment:
#include "memory.h"

And do not change this?

Reply:
Agreed

Reply:
I could, BUT keep in mind, this is in code that is ONLY for VC ;)